ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11-periodics.yaml

```diff
@@ -18792,193 +18792,6 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
-- agent: kubernetes
-  cluster: build03
-  cron: 32 20 16 2 *
-  decorate: true
-  decoration_config:
-    skip_cloning: true
-  extra_refs:
-  - base_ref: release-4.11
-    org: openshift
-    repo: openshift-tests-private
-  labels:
-    ci-operator.openshift.io/cloud: azure4
-    ci-operator.openshift.io/cloud-cluster-profile: azure-qe
-    ci-operator.openshift.io/variant: amd64-nightly-openshift-logging-5.5
-    ci.openshift.io/generator: prowgen
-    job-release: "4.11"
-    pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-openshift-logging-5.5-azure-ipi-fips-f360
```

Comment:
@anpingli I think there is no need to remove this job as it's frequency is every 360 days. Besides, if you must remove this job, please remove the job under `ci-operator/config/openshift/openshift-tests-private/`, then run `make jobs` or `make update` and push these changes to this PR.